lab_1/main.py

```diff
@@ -0,0 +1 @@
+hello = "k"
```

Comment:
You don't need this folder and this file. 
The laboratory work is supposed to be located in the `lab_1_classify_by_unigrams/main.py` file.

Reply:
I try to fix it all, i hope i will manage it soon